corehq/apps/api/resources/v0_1.py

```diff
@@ -270,7 +270,7 @@ class Meta(UserResource.Meta):
 
     def obj_get_list(self, bundle, **kwargs):
         domain = kwargs['domain']
-        username = bundle.request.GET.get('username')
+        username = bundle.request.GET.get('web_username')
```

Comment:
if this doesn't affect anyone who used the API since May 13 then I think it's fine from my side